libraries/helpers.php

```diff
@@ -9,4 +9,41 @@ function array_group($array, $key)
     }
 
     return $result;
-}
\ No newline at end of file
+}
+
+function welcome($msg1,$msg2)
```

Comment:
also let's move it to the console_helpers library

```suggestion
function info_box(...$lines)
```